aip/general/0203.md

```diff
@@ -44,6 +44,27 @@ document this behavior for clients.
 
 ## Vocabulary
 
+### Identifier
+
+The use of `IDENTIFIER` indicates that a field within a resource message is used
+to identify the resource. It **must** be attached to the `name` field and **must
+not** be attached to any other field (see [fields representing resource names]).
+
+This annotation **must not** be applied to references to other resources within
+a message.
+
+#### Identifier Backwards Compatibility
```

Comment:
We already have a `Backwards Compatibility` section for all interesting cases. Should this stuff go there?

Reply:
good call, will move.